src/components/Alert/AlertFieldLevel.tsx

```diff
@@ -1,9 +1,9 @@
 import type { JSXElement } from '~/src/types/jsxElement';
 import { Icon } from '../Icon/Icon';
 
-export type NotificationFieldLevelType = '' | 'error' | 'success' | 'warning';
+export type AlertFieldLevelType = '' | 'error' | 'success' | 'warning';
```

Comment:
Looks like a problem I introduced (sorry! 😬) but type='' should be replaced with type='info', otherwise we end up with an unusable UI.
<img width="320" alt="Screenshot 2023-12-01 at 11 07 23 AM" src="https://github.com/cfpb/design-system-react/assets/2592907/e6b3bec5-b23e-4c8d-8a88-724015a387b2">

```suggestion
export type AlertFieldLevelType = 'info' | 'error' | 'success' | 'warning';
```